src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -453,7 +447,7 @@ private static boolean hasEnoughLeavesAndIsSupposedToCut(@NotNull final IWorldRe
         {
             for (int dz = -1; dz <= 1; dz++)
             {
-                for (int dy = -1; dy <= 1 + dynamicBonusY; dy++)
+                for (int dy = -3; dy <= 1 + dynamicBonusY; dy++)
```

Comment:
Some mod trees may not enough leaves in 3x3x3 box centered at top log, so I broadened the search a bit more down

Reply:
Yeah, I had talked about this recently true